lib/index.js

```diff
@@ -33,6 +33,15 @@ var defaults = {
     encoding: 'utf8'
 };
 
+// Simple logger interace
+var logger = {
+    debug  : function() { console.log(arguments); },
+    verbose: function() { console.log(arguments); },
+    info   : function() { console.log(arguments); },
+    warn   : function() { console.log(arguments); },
+    error  : function() { console.log(arguments); }
+};
+
```

Comment:
Why you moved the logger above. I assume it has something todo with issue #105. But the you should create an own merge request for this.